src/TestUtils/src/Microsoft.Maui.IntegrationTests/TemplateTests.cs

```diff
@@ -1,19 +1,16 @@
 using Microsoft.Maui.IntegrationTests.Apple;
 
-[assembly: LevelOfParallelism(2)]
```

Comment:
Any chance we could try setting this to 1 to see if that stabilizes it? In theory that should bring it back to where it was from a functional perspective.

Reply:
I wanted to get this in to unblock PR's that are being branched from main. We can try that in a new PR.